app/front/scripts/directives/sidebar/filters/template.html

```diff
@@ -4,7 +4,7 @@
   <div ng-repeat="hierarchy in hierarchies" class="x-hide-first-separator"
     ng-if="selectedHierarchies.indexOf(hierarchy.key) == -1">
     <hr>
-    <div ng-repeat="filter in hierarchy.dimensions">
+    <div ng-repeat="filter in hierarchy.dimensions" ng-if="(filter.values.length > 1) && ($index <= lastFilterIndex[hierarchy.key] + 1)">
```

Comment:
Yes, I like it - and this will also accomplish the 'more than one value' filtering implicitly.

I only fear it will confuse users ("why can I see only levels 2 & 4? I want to filter by 3!").
So if we hide the filter, users won't know that there's only one value there.

Perhaps instead of hiding 1-value filters we can show them in a 'disabled' style?